// proto version
syntax = "proto3";

// proto package name
package HelloWorld;

// The request message containing the greeting text.
message HelloRequest {
    string text = 1;
}

// The response message containing the greetings.
message HelloReply {
    string message = 1;
}

// The Greeter service definition.
service Greeter {
    rpc SayHello(HelloRequest) returns (HelloReply) {}
}